zerver/actions/invites.py

```diff
@@ -206,6 +206,7 @@ def do_invite_users(
     user_profile: UserProfile,
     invitee_emails: Collection[str],
     streams: Collection[Stream],
+    send_notification: bool,
```

Comment:
Can we set a default value for this like we do for `invite_as` so that we do not have to pass this parameter in various places in tests? And we can only pass this where we need to test the `false` case.